docs/client/operations.md

```diff
@@ -0,0 +1,118 @@
+# <img align="center" src="../images/logo.png">  Calling Operations with Your Typescript Client
+
+AutoRest provides both synchronous and asynchronous method overloads for each service operation.
```

Comment:
Typescript only generates asynchronous methods

Reply:
Thanks! Had that everyewhere but didn't change the main intro. thanks!